tools/docker-receptor.yml

```diff
@@ -0,0 +1,60 @@
+#
+# a simple receptor network with a single control plane node and one hop
+# to nodes where receptor + runner are installed
+#
+# <awx:4444>
```

Comment:
This is the simplest useful topology that made sense to me in the development deployment.  We'll have to come up with some documented approach for building out and properly configuring production-grade clusters.  I expect at thing that users will want is example use cases and sample deployments, as well as in-depth narrative documentation on how to build out and troubleshoot networks across a variety of transports.